tests/Feature/ForumTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Tests\Feature;
 
+use App\Http\Requests\ThreadRequest;
```

Comment:
You can move the constant from the request to the test case. The reason why you should do this is because your test serves as a sort of contract which means you should try to be as explicit as possible for defining values instead of relying on imported constants. Say someone changes the value in the validator for some reason, you're never going to notice because it's inherited in the test. It's important for this test that the value *is* 60.

Reply:
Good point! I will make some changes ;)